### ISSUE TITLE
Ling Last Resort 

### ISSUE THREAD

**Issue by Swagile** [HippieStationCode/HippieStation13#3060]
When a ling last resorts, the blood can snipe you across an entire room and past windows.

Ling last resorted in escape shuttle and KO'd me through 4 reinforced windows.

This should not be intended.

**Comment by ArcLumin** [comment]
Yeah, it's if "you can see the slug", which is more of an oversight than anything.
